data_dir/cs_lwt_perf_int.yaml

```diff
@@ -0,0 +1,87 @@
+# Keyspace Name
+keyspace: cs_lwt_perf_int
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE cs_lwt_perf_int WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: targettabel
+
+# The CQL for creating a table you wish to stress (optional if it already exists)
+table_definition: |
+  CREATE TABLE targettabel (
+        pr1 timeuuid,
+        pr2 int,
+        pr3 smallint,
+        pr4 bigint,
+        pr5 double,
+        pr6 float,
+        pr7 tinyint,
+        pr8 decimal,
+        aux1 bigint,
+        aux2 bigint,
+        PRIMARY KEY(pr1, pr2, pr3, pr4, pr5, pr6, pr7, pr8)
+  ) WITH compaction = { 'class':'LeveledCompactionStrategy' }
+    AND comment='A table to hold blog posts'
+
+### Column Distribution Specifications ###
+
+columnspec:
+  - name: pr1
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr2
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr3
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr4
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr5
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr6
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr7
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr8
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux1
+    size: uniform(1..65534)
+    population: uniform(1..65534)
+  - name: aux2
+    size: uniform(1..65534)
+    population: uniform(1..65534)
+
+### Batch Ratio Distribution Specifications ###
+
+insert:
+  partitions: fixed(256)
```

Comment:
Why are we inserting such big batches?

Reply:
The idea here is not to have big batches, but to hit as many paritions as we can. This parameter is still to be investigated. At this point it seemse have 0 effect.